com.microsoft.mrtk.spatialmanipulation/Solvers/SolverHandler.cs

```diff
@@ -598,10 +596,8 @@ private bool IsInvalidTracking()
         /// </returns>
         private bool IsHandTracked(Handedness hand)
         {
-            Debug.Assert(HandSubsystem != null);
-
-            // todo - when HandAggregationSubsystem implements IsHandTracked, make that call here.
-            return HandSubsystem != null;
+            return XRSubsystemHelpers.HandsAggregator != null &&
+                   XRSubsystemHelpers.HandsAggregator.TryGetJoint(TrackedHandJoint.Palm, hand.ToXRNode().Value, out HandJointPose pose);
```

Comment:
We might want to cache and check that `hand.ToXRNode()` has a value first. If `Handedness.None` is passed in, this will null ref (or whatever the exception for calling `.Value` on null is)

Reply:
Done!